grails-app/services/org/bbop/apollo/SequenceService.groovy

```diff
@@ -217,29 +217,14 @@ class SequenceService {
     String loadResidueForSequence(Sequence sequence, int chunkNumber) {
         CRC32 crc = new CRC32();
         crc.update(sequence.name.getBytes());
-        String hex = String.format("%08x", crc.getValue());
-        String []dirs = splitStringByNumberOfCharacters(hex, 3);
+        String hex = String.format("%08x", crc.getValue())
+        String []dirs = hex.toList().collate( 3 )*.join()
```

Comment:
can you add a unit test to validate this.  I want to make sure we are accurately handling remainders, etc.